src/codegen/codegen_neuron_cpp_visitor.cpp

```diff
@@ -596,7 +596,7 @@ std::string CodegenNeuronCppVisitor::get_variable_name(const std::string& name,
     if (iter != info.neuron_global_variables.end()) {
         std::string ret;
         if (use_instance) {
-            ret = "*(inst->";
+            ret = "*(inst.";
```

Comment:
was that an error before?
by the way, why not use the `->` operator and drop the `*(...)` instead?

Reply:
ah.. duh.. yes obviously....